src/SpeedyWeather.jl

```diff
@@ -2,37 +2,37 @@ module SpeedyWeather
 
     # STRUCTURE
     import Parameters: @with_kw, @unpack
-    
+
     # NUMERICS
     import FastGaussQuadrature
     import AssociatedLegendrePolynomials
     import FFTW
     import Primes
     import LinearAlgebra
 
-    # INPUT OUTPUT 
+    # INPUT OUTPUT
     import Dates: Dates, DateTime
     import Printf: @sprintf
     import NetCDF: NetCDF, NcFile, NcDim, NcVar
     import BitInformation: round, round!
     import UnicodePlots
     import ProgressMeter
-    
+
     # EXPORT MAIN INTERFACE TO SPEEDY
     export run_speedy, initialize_speedy
 
     # EXPORT STRUCTS
-    export Parameters, GenLogisticCoefs,
+    export Parameters, GenLogisticCoefs, HumidityCoefs,
         GeoSpectral, Boundaries, Constants, Geometry, SpectralTransform,
         PrognosticVariables, DiagnosticVariables
-    
+
     # EXPORT SPECTRAL FUNCTIONS
     export  spectral, gridded,
         spectral_truncation, spectral_interpolation,
         triangular_truncation
 
     include("utility_functions.jl")
-    include("parameter_structs.jl")
```

Comment:
I've called it `parameter_structs` to have a place where structs can be defined before `default_parameters.jl` is included (otherwise you'll get an undefined type error). In think general there's many places to define "constants" which can serve different purposes
- parameters that can be changed through the `run_speedy` interface
- physical/numerical constants that should be defined as such for clarity and readability
- constants that have to be converted to a given number format NF to avoid runtime conversions

Reply:
Yeah sorry I was a bit hasty in renaming this file and had actually intended to discuss it with you!

In general I found the constants/parameters thing a bit confusing. As you suggest, I think it would be nice to differentiate somehow between universal physical constants that will never change and parameters which might.

Reply:
What about using "constants" for actual physical constants that nobody would want to change, "parameters" for everything that `run_speedy` may take as an argument and "constants_runtime" for every value that has to be precalculated, probably from parameters, but is constant at runtime. Meaning we should rename the constants.jl file and constants.jl should then be loaded before default_parameters?

Reply:
Yeah, I like that approach. I would write it up as a separate issue, just to avoid this PR becoming too bloated.